scrapbook-gateway-service/routes/album.py

```diff
@@ -17,32 +18,153 @@ def createAlbum():
     @return - http status code
```

Comment:
It doesnt only return the http status code, but also the created album.

if you're writing comments and documenting, please make sure it complies with the code.